.env

```diff
@@ -0,0 +1 @@
+BASE_URL=http://localhost:8080/api/v1/
```

Comment:
I think you need `VITE_` in front of `BASE_URL` this probably worked because there is `BASE_URL` in vite but it is used for something else